### ISSUE TITLE
Voice masks aren't updated for unassigned/spectators/hltv

### ISSUE THREAD

**Issue by WPMGPRoSToTeMa** [s1lentq/ReGameDLL_CS#5]
https://github.com/s1lentq/ReGameDLL_CS/blob/master/regamedll/game_shared/voice_gamemgr.cpp#L174
```C++
		if (!pEnt || !pEnt->IsPlayer())
			continue;
```
The problem is in the IsPlayer method:
https://github.com/s1lentq/ReGameDLL_CS/blob/master/regamedll/dlls/player.h#L496
```C++
	virtual BOOL IsPlayer(void)
	{
		return (pev->flags & FL_SPECTATOR) != FL_SPECTATOR;
	}
```

**Comment by s1lentq** [comment]
why do we need to update voice mask for spectator etc?
more please

**Comment by WPMGPRoSToTeMa** [comment]
@s1lentq because sv_alltalk cvar doesn't work properly with this check. When sv_alltalk 1 if mask is not updated, then spectators and hltv can't hear anyone, especially if this is the first connect for client with this index. If this is not first connect, then engine can use old voice masks.

**Comment by s1lentq** [comment]
i can see that, thanks for the explanation

**Comment by s1lentq** [comment]
Resolved #31

**Comment by s1lentq** [comment]
Resolved #31

**Comment by s1lentq** [comment]
fixes
